lib/truffle/securerandom.rb

```diff
@@ -56,6 +51,11 @@ module SecureRandom
   def self.random_bytes(n=nil)
     n ||= 16
 
+    begin
+      require 'openssl'
+    rescue LoadError # rubocop:disable Lint/HandleExceptions
+    end
```

Comment:
Since this is stdlib, could you add a `# Truffle:` comment to explain why we changed that and to make sure we don't revert it without noticing when updating the stdlib?
A test/spec would be even better to make sure it keeps working :)

Reply:
Do files in `lib/truffle` get updated when we update stdlib? I thought this was our own code.

I'm happy to add the comment, but I'm uncertain on the risk factor.

Reply:
Oh, I thought this file is `lib/mri/securerandom.rb`. I wonder why we don't use the MRI stdlib.
I think a test/spec would be valuable, so we would also catch it if e.g. it was a regression when using MRI's securerandom.